versions/2.0.md

```diff
@@ -1,26 +1,29 @@
 # OpenAPI Specification
-## (fka Swagger RESTful API Documentation Specification)
+## (Formerly-known-as the OpenAPI Specification)
 
 #### Version 2.0
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt).
 
-The Swagger specification is licensed under [The Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
+The OpenAPI specification is licensed under [The Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
 
 ## Introductions
 
-Swagger™  is a project used to describe and document RESTful APIs.
+The OpenAPI Specification is a project used to describe and document RESTful APIs. 
 
-The Swagger specification defines a set of files required to describe such an API. These files can then be used by the Swagger-UI project to display the API and Swagger-Codegen to generate clients in various languages. Additional utilities can also take advantage of the resulting files, such as testing tools.
+This project defines a set of files required to describe such an API. These files can then be used by tools to render API documentation or to generate client SDKs in various languages. Additional utilities can also take advantage of the resulting files, such as testing tools. 
+
+This is part of the OpenAPI Initiative (OAI), a collaboration between many companies under the auspices of the Linux Foundation. For more information on the initiative, its members, and its mandate, visit [OpenAPIs.org](http://openapis.org) The OpenAPI Specification grew out of the OpenAPI Specification that was donated to the OIA by SmartBear.
```

Comment:
Hmm, the last sentence is not really correct. If you are editing the 2.0 document, then it should describe the 2.0 document, and this didn't "grew out of something", but "is identical to the Swagger Specification", so people know that it is the same.

In a text for the next version something like your sentence (with a reference to Swagger) would be more fitting.